app/errors.py

```diff
@@ -0,0 +1,22 @@
+"""
+different messages processing
+was made in educational purposes
+"""
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, message: str = "NotWearingMaskError()") -> None:
+        super().__init__(message)
+
+
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return f"{self.__class__.__name__}()"

```

Comment:
I don't really understand this move, looks to complicated.

do smth like:
```
   def __init__(self, message: str):            
        # Call the base class constructor with the parameters it needs
        super().__init__(message)
 ```
 
 Exception already have message attribute so it will take it